apps/aecore/src/aec_headers.erl

```diff
@@ -26,24 +24,54 @@ time_in_secs(Header) ->
     Time = Header#header.time,
     aeu_time:msecs_to_secs(Time).
 
--spec serialize_for_network(header()) ->
```

Comment:
Kindly consult with others before making these API renaming changes. These should not be a unilateral decision! @sennui

Reply:
I had to make this change, because network parts wanted to use a function with different spec.